src/requirements/data/colleges/asPreFA2020.ts

```diff
@@ -119,6 +119,7 @@ const casPreFA2020Requirements: readonly CollegeOrMajorRequirement[] = [
       },
     },
   },
+  // TODO: AP/Test Credit cannot be applied to Distribution Requirements
```

Comment:
you might be able to use Ben's flag for `disallowTransferCredit`

Reply:
> Left some nits.
> 
> 1. A bigger issue I found was that the A&S [Fall 2020 and After] doesn't seem to have any majors connected to it. Idk if this is a fix you can do in the requirements data.
> 
>  Screen.Recording.2021-03-23.at.10.54.03.AM.mov

Oh yup i forgot to add them, fixing that. TY!

Reply:
and yes I added his flag in the other PR <3 removing comment now!